main/tasks/power_management_task.c

```diff
@@ -143,20 +143,21 @@ void POWER_MANAGEMENT_task(void * pvParameters)
     }
 }
 
-// all values input below 60 result in 38 but increases exponentially such that an input of 70 results in 100
+// Fan speed between 33 min and 100 max based on input,
+// proportional to temp increase between 50 and THROTTLE_TEMP
 static void automatic_fan_speed(float chip_temp)
 {
     double result = 0.0;
-    double min_temp = 50.0;
+    double min_temp = 40.0;
 
     if (chip_temp < min_temp) {
-        result = 25;
+        result = 33;
     } else if (chip_temp >= THROTTLE_TEMP) {
         result = 100;
     } else {
         double range = THROTTLE_TEMP - min_temp;
-        result = ((chip_temp - min_temp) / range) * 100;
+        result = ((chip_temp - min_temp) / range) * 67 + 33;
```

Comment:
maybe define 67 as fan_range and set it equal to 100 - minimum_fan_speed.

Reply:
Done